CONTRIBUTING.md

```diff
@@ -1,16 +1,281 @@
+
+### General rules
+
+* Before opening a new Pull Request, have a look at
+  [existing ones][existing-pull-requests]. Maybe someone has already opened one
+  about the same thing. If it's the case, you might be able to help with the
+  contribution. Just comment on the PR and ask.
+  Old and stalled [PRs are sometimes archived][archived-pull-requests] with a
+  "memo" label, maybe one of them is also about the same topic.
+
+* Each Pull Request form uses a template with 3 sections that are here to help
+  maintainer understand your contribution and help them testing it:
+
+      #### Contribution description
+
+      #### Testing procedure
+
+      #### Issues/PRs references
+
+  Please fill each section with as much information as possible.
+
+* The Pull Request title should reflect what it is about and be in the form:
+
+  `area of change: description of changes`
+
+* Remember that smaller PRs tend to be merged faster, so keep your changes as
+  concise as possible. They should be confined to a single explainable
+  change, and be runnable on their own. So don't hesitate to split your PRs
+  into smaller ones when possible.
+
+* In the Pull Request form, we recommend that you leave the
+  "Allow edits from maintainers" check box ticked. This will allow maintainer
+  finalizing your PR by pushing in your branch. In general, this speeds up the
+  PR merge in the main repository. Note that this is not an obligation.
+
+* Check if your code follows the [coding conventions][coding-conventions]. If
+  it doesn't, you can [uncrustify][uncrustify] a file:
+
+      $ uncrustify -c $RIOTBASE/uncrustify-riot.cfg <your file>
+
+* RIOT provides static test tools to verify the quality of changes (cppcheck,
+  trailing whitespaces, documentation, etc). These tools are wrapped in a
+  single `make` target: `static-test`.
+
+  *Watch out:* the command below will rebase your branch on your master branch,
+  so make sure they can be rebased (e.g. there's no potential conflict).
+
+      $ make static-test
+
+  Use it before opening a PR to perform last time checks.
+
+* Each commit should target changes of specific parts/modules of RIOT. The
+  commits use the following pattern:
+
+  `area of code: description of changes`.
+
+  You can use multi-line commit messages if you want to detail more the
+  changes.
+
+* Try to answer reviews as quickly as possible to speed up the review process
+  and avoid stalled PRs.
+
+* Maintainers try their best to review every PR as fast as possible, but they
+  are also only human and it can happen that they miss a few PRs or might be
+  preoccupied with other PRs. If it happens that your PR receive no review for
+  a long time, don't hesitate to gently solicit a review by commenting or
+  by explicitly mentionning a maintainer that you know is knowledgeable in the
+  area of the PR. You can also advertise the PR on the devel mailing list and
```

Comment:
Maybe add a link to http://lists.riot-os.org/mailman/listinfo/devel or a "mailto:devel@riot-os.org" link?